### PR TITLE
[inductor] Fix alignment issue with new runtime

### PR DESCRIPTION
**Pull request by jansel**
This fixes a minor issue in #1338 that only shows up in one internal model.

### CODE REVIEW

**Comment by soumith** [comment]
add a test?

**Comment by jansel** [comment]
> add a test?

I was trying to make a more minimal repro for this but having a hard time.  Passing the wrong alignment hints to Triton don't always produce invalid code.

File: torchinductor/codegen/triton.py [hunk 1]
```diff
@@ -47,14 +47,15 @@ def signature_of(arg):
...
+        assert isinstance(x, SizeArg)
+        return V.graph.sizevars.maybe_guard_multiple_of(x.expr, ALIGNMENT)
 
-    divisible_by_16 = [
-        i
-        for i, arg in enumerate(args)
-        if isinstance(arg, TensorArg)
-        or V.graph.sizevars.maybe_guard_multiple_of(arg.expr, JITFunction.divisibility)
-    ]
+    divisible_by_16 = [i for i, arg in enumerate(args) if is_aligned(arg)]
```
Comment (ngimel) [inline torchinductor/codegen/triton.py:58]: lol the place I worried about turned out to be problematic

**Review by ngimel — APPROVED** [review]

File: torchinductor/codegen/triton.py (hunk 1 above)
Comment (jansel) [reply torchinductor/codegen/triton.py:58]: Yeah it did :)

**Pull request merged**